src/client/gelbooru.rs

```diff
@@ -1,22 +1,26 @@
-use super::{ClientBuilder, ClientType};
+use async_trait::async_trait;
+
+use super::{Client, ClientBuilder};
 use crate::model::gelbooru::*;
 
 /// Client that sends requests to the Gelbooru API to retrieve the data.
-pub struct GelbooruClient(ClientBuilder);
+pub struct GelbooruClient(ClientBuilder<Self>);
 
-impl From<ClientBuilder> for GelbooruClient {
-    fn from(value: ClientBuilder) -> Self {
+impl From<ClientBuilder<Self>> for GelbooruClient {
+    fn from(value: ClientBuilder<Self>) -> Self {
         Self(value)
     }
 }
 
-impl GelbooruClient {
-    pub fn builder() -> ClientBuilder {
-        ClientBuilder::new(ClientType::Gelbooru)
-    }
+#[async_trait]
+impl Client for GelbooruClient {
+    type Post = GelbooruPost;
+
+    const URL: &'static str = "https://gelbooru.com";
+    const SORT: &'static str = "sort:random";
```

Comment:
The SORT associated constant here and in the Danbooru client also need fixing. It should be just `"sort:"`